DEVELOPMENT.md

```diff
@@ -113,13 +113,13 @@ While iterating on the project, you may need to:
 To make changes to these CRDs, you will probably interact with:
 
 * The CRD type definitions in [./pkg/apis/pipeline/alpha1](./pkg/apis/pipeline/v1alpha1)
-* The controllers in [./pkg/controller](./pkg/controller)
-* The clients are in [./pkg/client](./pkg/controller) (these are generated by `./hack/update-codegen.sh`)
+* The controllers in [./cnd/controller](./cmd/controller)
```

Comment:
This should likely be `reconcilers in [./pkg/reconciler](./pkg/reconciler)`. The actual `cmd/controller` package doesn't get touched too much.

Reply:
Fixing - I was just looking for valid links.